base/Terminals.jl

```diff
@@ -156,7 +156,11 @@ else
     function hascolor(t::TTYTerminal)
         startswith(t.term_type, "xterm") && return true
         try
-            return success(`tput setaf 0`)
+            @static if Sys.KERNEL == :FreeBSD
```

Comment:
It's only for FreeBSD.

I tried OpenBSD, it's more complicate:
```
# TERM=rxvt tput setaf 1 && echo 'work'  
work
# TERM=rxvt-256color tput setaf 1 && echo 'work'
tput: not enough arguments (3) for capability `setaf'
# TERM=rxvt-256color tput setaf 1 0 0 && echo 'work'
work
# TERM=screen tput setaf 1 && echo 'work'             
work
# TERM=screen-256color tput setaf 1 && echo 'work'    
tput: not enough arguments (3) for capability `setaf'
# TERM=screen-256color tput setaf 1 0 0 && echo 'work'
work
# uname -a                                                                     
OpenBSD obsd.my.domain 6.0 GENERIC.MP#2319 amd64
```

In case of NetBSD, both `tput AF` and `tput setaf` work:
```
# uname -a
NetBSD  7.0.2 NetBSD 7.0.2 (GENERIC.201610210724Z) amd64
# TERM=rxvt tput setaf 1 && echo 'work'
work
# TERM=rxvt-256color tput setaf 1 && echo 'work'
work
# TERM=screen tput setaf 1 && echo 'work'
work
# TERM=screen-256color tput setaf 1 && echo 'work'
work
# TERM=screen-256color tput AF 1 && echo 'work'
work
```

In case of DragonFly... seems subtle. `tput setaf` not work, and `tput AF` only works on non `-256color` term type.